sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ServiceAPITest.groovy

```diff
@@ -270,6 +270,22 @@ class ServiceAPITest extends APISpec {
         primaryBlobServiceClient.setPropertiesWithResponse(sentProperties, null, null).getStatusCode() == 202
     }
 
+    def "Set props cors check"() {
+        setup:
+        def serviceProperties = primaryBlobServiceClient.getProperties()
+
+        def rule = new BlobCorsRule()
+        rule.setAllowedOrigins("microsoft.com")
+        rule.setMaxAgeInSeconds(60)
+        rule.setAllowedMethods("GET")
+        rule.setAllowedHeaders("x-ms-version")
+
+        serviceProperties.setCors(Collections.singletonList(rule))
+
```

Comment:
Can you leave a comment calling out that some properties are not set and the test validates that we fill them with null? Otherwise it'll be easy to forget what this test is for.